pkg/util/helpers.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright 2023 The KEDA Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+// Contains checks if a slice contains a given element.
+func Contains[T comparable](s []T, e T) bool {
+	for _, v := range s {
+		if v == e {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
I know that it's a small function, but a unit test here would be nice for the future

Reply:
Sure Jorge.

I saw a lot of helpers written in individual files, I didn't touch those for now but later on we can move all those here and generalize them if possible.